src/components/atoms/Icons/Wallet.tsx

```diff
@@ -4,6 +4,7 @@ import {SvgIcon, SvgIconProps} from '@material-ui/core';
 
 import Polkadot24 from 'src/images/network/Polkadot24.svg';
 import CoinbaseDisabled from 'src/images/wallet/coinbase-disabled.svg';
+import CommontWallet from 'src/images/wallet/commonWallet.svg';
```

Comment:
please change from `Commont` to `Common`

Reply:
resolve in https://github.com/myriadsocial/myriad-web/pull/1515/commits/c1d083d95c731d2f4b2b5a0fff1f7acc652debed